module_overviews/be_mod2.md

```diff
@@ -16,14 +16,13 @@ Creating database-backed applications in Rails, utilizing MVC pattern, testing u
 
 ## Projects
 
-<section>
-  <a href="https://backend.turing.io/module2/projects/index" class="btn btn-dark">Back End M2 Projects</a>
-</section>
+<ul class="projects">
+  <li class="project"><a href="https://backend.turing.edu/module2/projects/relational_rails">Relational Rails</a></li>
```

Comment:
I _think_ we have to add styles for this somewhere. Not sure if they already exists, but it would be something like this in one of the SCSS files:

```css
.project a:link,
.project a:visited,
.project {
  color: darken($dark-gray, 10%);
  text-decoration: none;
  font-weight: 500;
}
```